Configure

```diff
@@ -993,6 +993,25 @@ $target{build_scheme} = [ $target{build_scheme} ]
 my ($builder, $builder_platform, @builder_opts) =
     @{$target{build_scheme}};
 
+foreach my $checker (($builder_platform."-".$target{build_file}."-checker.pm",
+                      $builder_platform."-checker.pm")) {
+    my $checker_path = catfile($srcdir, "Configurations", $checker);
+    if (-f $checker_path) {
+        my $fn = $ENV{CONFIGURE_CHECKER_WARN}
```

Comment:
It's for the "I fscking know what I'm doing" mode.  For example, it happens that I do `./Configure VC-WIN32` on my Linux box just to check certain aspects of `configdata.pm` without having to start a VirtualBox instance runnign Windows...